src/main/java/dk/kb/image/ProxyHelper.java

```diff
@@ -166,6 +167,12 @@ private static HttpURLConnection establishConnection(String request, URI uri, UR
             throw new InternalServiceException("Unable to set 'GET' as request method for '" + request + "'");
         }
         connection.setRequestProperty("User-Agent", "ds-image");
+     
+        if (httpHeaders != null) { //The test code will call it without httpHeaders
+          connection.addRequestProperty("Accept",httpHeaders.getHeaderString("Accept"));
+          log.info("Transfering accept header to image server:"+httpHeaders.getHeaderString("Accept"));
```

Comment:
Added to log on line 71 (debug)

Reply:
The logline is now
```Java
    	log.debug("proxy(request='{}', uri='{}', clientRequestURI='{}', httpServletResponse={}, acceptHeader{}) called",
                  request, uri, clientRequestURI, httpServletResponse == null ? "not present" : "present", acceptHeader);
```
This will generate an output such as
```
    request='...', ..., httpServletResponse=not present, acceptHeaderimage/png
```
Please ensure that the output is easily readable (`acceptHeaderimage/png` is not).

Reply:
missed it, even from reading log file

FIXED!